persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_1/generated/persist_types.bal

```diff
@@ -41,6 +48,9 @@ public type EmployeeUpdate record {|
     time:Date birthDate?;
     string gender?;
     time:Date hireDate?;
+    @constraint:String {
```

Comment:
In this example shouldn't we add the constraint for all the records, In workspace and building records, the constraint is missing

Reply:
We don't need to add to all the string fields. If the user wants to change the length, they may add this annotation.

Reply:
workspace and building records don't have any constraint annotation in the model file. So, We don't add it here too.